cumulusci/files/templates/project/scratch_def.json

```diff
@@ -1,10 +1,13 @@
 {
   "orgName": "{{ package_name }} - {{ org_name }}",
   "edition": "{{ edition }}",
-  "orgPreferences" : {
-    "enabled": [
-      "S1DesktopEnabled",
-      "ChatterEnabled"
-    ]
+  "settings": {
+    "orgPreferenceSettings": {
+      "s1DesktopEnabled": true
+      , "chatterEnabled": true
+      {% if not managed %}
+      , "translation": true 
```

Comment:
I'm not a fan of the commas at the start of the line, but I see how it helps with the conditional and I'm willing to put up with it.

Reply:
I don't love it either but it seemed less messy than the alternative conditional style.